api/src/services/import-services/utils/datetime.test.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { formatTimeString } from './datetime';
+import { areDatesEqual, formatTimeString } from './datetime';
 
 describe('formatTimeString', () => {
```

Comment:
Could add a unit test for when the input is `null`

Reply:
roger